tests/unit/test_extensionPoints.py

```diff
@@ -48,56 +48,259 @@ def test_isWeak(self):
 		self.assertIsNone(method)
 
 class TestCallWithSupportedKwargs(unittest.TestCase):
+	"""
+	Tests to ensure that the correct parameters/kwargs are passed through to the handler method.

```

Comment:
I'm a bit worried about the term `kwargs` throughout. It's not clear to me when you mean capturing extra kwargs (`**kwargs`) vs named arguments (`a=1`). Perhaps you could use `**kwargs` to be explicit... or the term `extra kwargs` maybe?